errors/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports.customErrors = (error, request, respond, next) => {}
```

Comment:
I know why we have set these up already, but our code should be driven by tests and our commits only related to the current task. So please remove these until you need them.